services/gtfs_service.go

```
@@ -0,0 +1,548 @@
+		}
+		if len(next) > 1 {
+			item.NextDeparture2 = next[1].Format(departureDatetimeLayout)
+		}
+		result = append(result, item)
+	}
+
+	return result, nil
+}
+
+func (s *GTFSService) downloadAndParse() (*gtfsData, error) {
+	resp, err := http.Get(s.gtfsURL) // #nosec G107
```

golangci-lint (2.12.2)

[error] net/http.Get must not be called. use net/http.NewRequestWithContext and (*net/http.Client).Do(*http.Request)

(noctx)